packages/fxa-settings/src/pages/Signin/SigninBounced/en.ftl

```diff
@@ -1,7 +1,7 @@
 signin-bounced-header = Sorry. We’ve locked your account.
 # $email (string) - The user's email.
 signin-bounced-message = The confirmation email we sent to { $email } was returned and we’ve locked your account to protect your { -brand-firefox } data.
-# linkExternal is a link to a mozilla support
-signin-bounced-help = If this is a valid email address, <linkExternal>let us know</linkExternal> and we can help unlock your account.
+# supportButton is button which logs the user's action and navigates them to mozilla support
+signin-bounced-help = If this is a valid email address, <supportButton>let us know</supportButton> and we can help unlock your account.
```

Comment:
Is it necessary to change `<linkExternal>` here? If so, we'll need to update the ID for `signin-bounced-help` to trigger updates for the tag change.

Reply:
ohhh I see -- the HTML component which is being used had to be changed, so I changed the name to be reflective of that, but I can leave it as it was (it does still function as if it were an external link, it just also logs some metrics before redirecting the user.) Does changing the HTML component  need to trigger changes (if we don't change the variable?)

Reply:
The short answer is, if its purely a change on the backend and on the there doesn't appear to be any change to the end user, then we wouldn't need to update the string.

If the context as it appears to the end-user doesn't change - as in `<linkExternal>`/`<supportButton>` essentially look/function the same on the surface, then we wouldn't need to trigger changes. However, if for example the change here makes the text look like a button instead of a link, then perhaps it would be wise to update the variable and ID.

Variable name changes for existing strings end up requiring a new translation (but text is essentially the same as before), though likely a fairly simple update it still ends up requiring a translator to look at. For some locales, it could ultimately end up in fallback text appearing for some time until a team has the bandwidth to look at it.

Reply:
I'll just set the var back to what it was in that case -- there will be no change to the appearance of the component on the page

Reply:
Thank you @bcolsson !